docs/source/user-guide/common-operations/expressions.rst

```diff
@@ -110,6 +110,35 @@ This function returns an integer indicating the total number of elements in the
 
 In this example, the `num_elements` column will contain `3` for both rows.
 
+To concatenate two arrays, you can use the function :py:func:`datafusion.functions.list_cat` or :py:func:`datafusion.functions.list_concat`.
+These functions return a new array that is the concatenation of the input arrays.
```

Comment:
I recommend we change all references in this file to the `array_` versions rather than the `list_` versions since I think the primary method is `array_` and the `list_` version is a convenient alias.

Reply:
I have changed the list_ references to array_ references.

Reply:
Thank you! I’ll wait for CI to pass and then I’ll merge. Nice work on this PR!